usb/usb.go

```diff
@@ -190,7 +190,7 @@ func goBuildDynamic() error {
 		}
 		args = append(args, "-files", filepath.Join(workingDir, v)+":"+v)
 	}
-	args = append(args, "-files", "html")
+	args = append(args, "-files", "pkg/sos/html")
```

Comment:
if you still want nichrome to look for html in /, you can do pkg/sos/html:html I think. But I think it's best not to rely on that.